tests/conftest.py

```diff
@@ -60,8 +58,20 @@ def get_model(name):
         return po.metric.naive.mse
     elif name == 'ColorModel':
         return ColorModel().to(DEVICE)
-    elif name == 'FrontEnd':
-        return po.simul.FrontEnd(pretrained=True, requires_grad=False).to(DEVICE)
+
+    # FrontEnd models:
```

Comment:
looks like the only tests for the `frontend` models is with the (31, 31) size. I think we should probably test other shapes and their learnability

Reply:
The beauty of these implementations is that the learnable parameters are independent of kernel size, since all we're doing is learning gaussian stdevs and some scalars.

I'm instead going to test the shape and other arguments of `gaussian1d` and `circular_gaussian2d`, the functions that actually create the conv kernels.